gui/app/controls/readings/TvDisplay.qml

```diff
@@ -10,5 +10,5 @@ ParameterDisplay {
     // show *max* volume averaged over the past few breaths?  This
     // is what "TV" actually means, and I suspect it's much more
     // meaningful than showing the last-measured patient volume.
-    parameterValue: GuiStateContainer.measured_tv.toFixed(0);
+    parameterValue: (Math.round(GuiStateContainer.measured_tv / 10) * 10).toFixed(0)
```

Comment:
If we're doing toFixed(0) I'm unclear on why we also round it?  Unless you specifically want Math.round's behavior (round to +inf) instead of toFixed's behavior (round to next odd, I still cannot believe I have that correct).